packages/web-react/src/hooks/__tests__/useResizeObserver.test.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable jest/no-disabled-tests */
```

Comment:
It would be great imho to link an issue if we already have for these additional tests. And if we don't, create one.

Reply:
Issue added: https://jira.almacareer.tech/browse/DS-1796